store/src/java/com/zimbra/cs/service/mail/ToXML.java

```diff
@@ -2295,7 +2295,8 @@ public static Element encodeInviteComponent(Element parent, ItemIdFormatter ifmt
                     descElem.setText(desc);
                 }
                 String descHtml = invite.getDescriptionHtml();
-                BrowserDefang defanger = DefangFactory.getDefanger(MimeConstants.CT_TEXT_HTML);
+                boolean isOwasp = Provisioning.getInstance().getConfig().getBooleanAttr(Provisioning.A_zimbraUseOwaspHtmlSanitizer, Boolean.TRUE);
```

Comment:
Please enhance the DefangFactory.getDefanger() to check the configuration and accordingly return the defanger.

Reply:
done